e2e/testconfig/testconfig.go

```diff
@@ -190,42 +187,64 @@ var govGenesisFeatureReleases = semverutil.FeatureReleases{
 
 // defaultModifyGenesis will only modify governance params to ensure the voting period and minimum deposit
 // are functional for e2e testing purposes.
+// Note: this function intentionally does not use the type defined here https://github.com/tendermint/tendermint/blob/64747b2b184184ecba4f4bffc54ffbcb47cfbcb0/types/genesis.go#L39
```

Comment:
should we link to a release tag instead of a commit?

Reply:
Yeah I think that's a good idea, will update the comment!